packages/trader/src/sass/app/_common/components/contract-type-info.scss

```diff
@@ -144,17 +169,51 @@
     }
     &__icon {
         cursor: pointer;
+        padding-right: 2.4rem;
     }
     &__scrollbars {
         margin-top: 2.4rem;
         margin-bottom: auto;
+        &--glossary {
+            margin-top: 0;
+        }
+
+        @include mobile {
+            margin-top: 1.6rem;
+        }
     }
     &__button {
         margin-top: 2.4rem;
         @include mobile {
-            width: 27.2rem;
+            width: 100%;
             padding: 1.5rem 0;
             align-self: center;
         }
     }
+    &__button-wrapper {
+        background: var(--general-section-1);
+        padding: 0.4rem;
+        margin: 1.6rem;
+        border-radius: 0.6rem;
+
+        @include mobile {
+            width: 32.8rem;
+            margin: 1.6rem auto;
+        }
+
+        .dc-button-menu__button {
+            background-color: var(--general-section-1) !important;
```

Comment:
Is there a chance to write more specific css selector to avoid using `!important` in this whole section?

Reply:
no. because its a shared component and i cant target it with id since the `--active` state is being toggled.

Reply:
Won't it break the styles anywhere else since it's a shared component?